profiles/models.py

```diff
@@ -142,6 +180,8 @@ class Profile(models.Model):
     linkedin = JSONField(blank=True, null=True)
     student_id = models.IntegerField(blank=True, null=True, unique=True)
 
+    date_updated = models.DateTimeField(blank=True, null=True, auto_now=True)
```

Comment:
I wasn't aware of that, but if we have a standard way of doing this I'm all in favor of it. I'll update with that change.